docs/source/index.rst

```diff
@@ -74,18 +74,23 @@ Common
 
 - Start mavros and VRPN for mocap: ```roslaunch mslquad quad_vrpn.launch```
 
-- May need to change ```vrpn_server_ip``` in ```launch/vrpn.launch``` to the ip address of the mocap machine.
+- May need to change ```vrpn_server_ip``` in ```launch/vrpn.launch``` to the ip
+  address of the mocap machine.
```

Comment:
change mocap to `mocap` you shouldn't need to hard code the ip adress `mocal.local` should resolve

Reply:
Just mocap or mocap.local?

Reply:
the launch file should have `mocal.local`